packages/translate/lib/testLine.tsx

```diff
@@ -89,11 +89,15 @@ const translateAssertTestLine = (
 	elements?: Elements,
 	lineResult?: TestLineResult,
 ): TestLineNode => {
+	let status = lineResult?.result;
+	if (testLine.then === 'fail' && status) {
```

Comment:
How about "then warning", "then exit"?

Reply:
I thought they should work like it does now? or do you mean to add ut for these cases?

Reply:
Example:
```
‼ Assert application has exited timeout 2s then warn
  warning: Condition not met
```

```
» Assert application has exited timeout 2s then exit
  exit: Condition was met
```